src/content/concepts/loaders.mdx

```diff
@@ -12,6 +12,14 @@ contributors:
   - byzyk
   - debs-obrien
   - EugeneHlushko
+  - wizardofhogwarts
+related:
```

Comment:
Since you've fixed the links, this section is not needed anymore :(

Reply:
@EugeneHlushko can we just keep references at one place for further reading? What do you think?

Reply:
IMO, links had good placement in their context, as a fix to non working block it looked like a good idea to me, right now, it looks wrong as from a guide about loaders you link to three specific ones as a read more. Despite us having a link to loaders in the header 
<img width="1011" alt="Screen Shot 2019-03-25 at 11 09 37 AM" src="https://user-images.githubusercontent.com/10549495/54907580-8611b080-4eee-11e9-96f7-2ab305f259e8.png">

Reply:
@EugeneHlushko related section removed